README.md

```diff
@@ -4,6 +4,25 @@ This is an open API specification for issuing digital credentials based on stand
 
 The latest version can be viewed at https://w3c-ccg.github.io/vc-issuer-http-api/index.html.
 
+## Versioning
+
+This repo and specification use [Semantic Versioning 2.0.0](https://semver.org/#semantic-versioning-200).
+
+The API version is defined [api.yml](./api.yml).
+
+You can browse the [latest release of github](https://github.com/w3c-ccg/vc-issuer-http-api/releases).
+
+When implementing this specification servers must respect the `VC-API-Version` HTTP Header. 
```

Comment:
Digital Bazaar would prefer to have versioning defined in this order:

* A version that is passed in through the POST body -- `options.version`
* A version that is passed in as a query parameter (works for POST, GET, and implementations that don't have access to Headers)
* A version that is passed in via HTTP Header (this proposal)

If a version cannot be determined:
* Version 1.0 is assumed, always.
* An upgrade may be performed if there is a deterministic algorithm to detect a minimum version... for example, if we add `options.sparklyPants` in version 2.0... then the implementation knows that the minimum version is 2.0 instead of 1.0.

Reply:
Why not have the version as part of the URL path? (as in the VC Data Model examples)